modules/combined/tests/gap_heat_transfer_htonly/tests

```diff
@@ -31,4 +31,24 @@
     compiler = 'CLANG GCC'
   [../]
 
+  [./cyl3D]
+    type = Exodiff
+    input = 'cyl3D.i'
+    exodiff = 'cyl3D_out.e'
+  [../]
```

Comment:
Each of these tests needs a `superlu = true` line for people who don't have that package available